source/issue/show-hide-state-not-set/index.html.md

```diff
@@ -44,8 +44,9 @@ Use a screen reader. Testing tools don't find this.
 ## How to fix it
 
 Move the `aria-expanded` attribute to the element which triggers the show/hide functionality. Although it is [according to spec](https://www.w3.org/TR/wai-aria-1.1/#aria-expanded), it doesn't work with any of the major screen readers.
```

Comment:
"Although this complies with the xyz specification, it does not work with any of the major screen readers.